src/interaction/selection.ts

```diff
@@ -27,15 +28,16 @@ function recompute(): void {
     // can result in null anchorNode and focusNode if there was no previous selection on page
     if (current.anchorNode === null && current.focusNode === null) { return; }
 
-    if (previous !== null && data.start !== null && data.start !== current.anchorNode) {
+    let startNode = data.start ? (data.start as TargetInfo).node : null;
```

Comment:
>startNode [](start = 8, length = 9)

Knowing the start and end elements isn't really that useful. Can you send the complete list of selected elements instead?

Reply:
Leaving this out of scope for now. This is on my list but it will be a slightly bigger change. Will get to it in next version.

---
In reply to: [368678337](https://github.com/microsoft/clarity-js/pull/166#discussion_r368678337) [](ancestors = 368678337)